db/db_impl_compaction_flush.cc

```diff
@@ -205,6 +205,8 @@ Status DBImpl::FlushMemTablesToOutputFiles(
     const autovector<BGFlushArg>& bg_flush_args, bool* made_progress,
     JobContext* job_context, LogBuffer* log_buffer) {
   Status s;
+  int count = 0;
```

Comment:
size_t

Reply:
Column family ID is `uint32_t`, it's very, very rare that we need `size_t` here.